business/checkers/virtual_services/subset_presence_checker.go

```diff
@@ -47,10 +47,6 @@ func (checker SubsetPresenceChecker) Check() ([]*models.IstioCheck, bool) {
 			for destWeightIdx := 0; destWeightIdx < destinationWeights.Len(); destWeightIdx++ {
 				destinationWeight, ok := destinationWeights.Index(destWeightIdx).Interface().(map[string]interface{})
 				if !ok || destinationWeight["destination"] == nil {
-					valid = false
```

Comment:
Mmm, how this logic affects to the gateways ?

Reply:
This validation is covered by galley. So it is not necessary to be here anymore.
The validation was already removed from documentation before. So no more changes need on that side.